config/locales/javascript/javascript.en.yml

```diff
@@ -239,9 +239,9 @@ en:
       contacts: "Contacts"
       posts: "Posts"
 
-    conversation:
-      new:
-        no_contacts: "You need to add some contacts before you can start a conversation."
```

Comment:
Yes. That's because that message:

![](https://cloud.githubusercontent.com/assets/3798614/18817159/461d94f6-835a-11e6-8922-4df169b142eb.png)

was previously rendered client-side ([here](https://github.com/diaspora/diaspora/pull/7108/files#diff-bf05c23497eed4643bb8fe3148aa9f47L39)) and is now rendered server-side ([here](https://github.com/diaspora/diaspora/pull/7108/files#diff-bd1e796410e7b0390610775a446df10aR32)) and I can't use Javascript translation key from Ruby code (I tried).